cmd/vulkan_sample/main.cpp

```diff
@@ -377,12 +377,34 @@ uint32_t inline GetMemoryIndex(
   return memory_index;
 }
 
+void usage() {
+  std::cout << "Options: \n";
+  std::cout << "-h=<height> Set desktop window heigth (default: 768)\n";
+  std::cout << "-w=<width>  Set desktop window width (default: 1024)\n";
+}
+
 #ifdef __ANDROID__
 int main_impl() {
 #else
 int main(int argc, const char** argv) {
 #endif
-  if (!CreateNativeWindow()) {
+  int width = 1024;
+  int height = 768;
+
+  for (int i = 1; i < argc; i++) {
+    std::string arg(argv[i]);
+    if (arg.compare(0, 3, "-h=") == 0) {
+      height = std::stoi(&(arg[3]));
+    } else if (arg.compare(0, 3, "-w=") == 0) {
+      width = std::stoi(&(arg[3]));
+    } else {
+      std::cout << "Unrecognized argument: " << arg << '\n';
+      usage();
+      std::exit(EXIT_FAILURE);
```

Comment:
To be consistent with the rest of the application, return -1

Reply:
Done